tests/protocol/test_binary.py

```diff
@@ -238,7 +238,7 @@ def reader_writer_ids(x):
     ], vset(TType.BOOL, vbool(True), vbool(False), vbool(True))),
 
     # list = vtype:1 count:4 (value){count}
-    (TType.LIST, [0x0C, 0x00, 0x00, 0x00, 0x00], vset(TType.STRUCT)),
+    (TType.LIST, [0x0C, 0x00, 0x00, 0x00, 0x00], vlist(TType.STRUCT)),
```

Comment:
A test utility function to create a `ListValue`. The previous test was wrong because it was trying to basically do `ListValue(..) == SetValue(..)`. It passed because the `__eq__` method didn't check classes, only attributes (which do match between list and set). With cython's `richcmp`, we are matching types too.